api/api/views/media_views.py

```diff
@@ -58,15 +65,30 @@ def get_queryset(self):
             ).values_list("provider_identifier")
         )
 
-    def get_serializer_context(self):
+    def get_serializer_context(
+        self, strategy: Literal["search", "collection"] = "search"
+    ):
         context = super().get_serializer_context()
-        req_serializer = self._get_request_serializer(self.request)
+        req_serializer = self._get_request_serializer(self.request, strategy)
         context.update({"validated_data": req_serializer.validated_data})
         return context
 
-    def _get_request_serializer(self, request):
-        req_serializer = self.query_serializer_class(
-            data=request.query_params, context={"request": request}
+    def _get_request_serializer(
+        self,
+        request,
+        strategy: Literal["search", "collection"],
```

Comment:
Extracted in [1f2a351](https://github.com/WordPress/openverse/pull/2853/commits/1f2a351b184af51ea630cf7291c822a8e53800c4). I'm not sure about the name, though. It is what the `strategy` for search view is set to, but since it describes two kinds of lists that we can get for media, I called it `ListView`. If you have a better suggestion, I would be happy to update.